system-variables.md

```diff
@@ -1566,73 +1559,63 @@ SET tidb_slow_log_threshold = 200;
 - Default value: ""
 - This variable is used to set the time point at which the data is read by the session. For example, when you set the variable to "2017-11-11 20:20:20" or a TSO number like "400036290571534337", the current session reads the data of this moment.
 
-### tidb_stats_load_sync_wait <span class="version-mark">New in v5.4.0</span>
+### tidb_stats_load_pseudo_timeout <span class="version-mark">New in v5.4.0</span>
 
-> **WARNING:**
+> **Warning:**
 >
 > Currently, synchronously loading statistics is an experimental feature. It is not recommended that you use it in production environments.
 
-- Scope: SESSION | GLOBAL
-- Default value: `0`
-- Unit: milliseconds
-- Range: `[0, 4294967295]`
-- This variable controls whether to enable the synchronously loading statistics feature. The default value `0` means that the feature is disabled. To enable the feature, you can set this variable to a timeout (in milliseconds) that SQL optimization can wait for at most to synchronously load complete column statistics. For details, see [Load statistics](/statistics.md#load-statistics).
+- Scope: GLOBAL
+- Default value: `OFF`
+- This variable controls how TiDB behaves when the waiting time of SQL optimization reaches the timeout to synchronously load complete column statistics. The default value `OFF` means that SQL execution fails after the timeout. If you set this variable to `ON`, the SQL optimization gets back to using pseudo statistics after the timeout.
 
-### `tidb_stats_load_pseudo_timeout` <span class="version-mark">New in v5.4.0</span>
+### tidb_stats_load_sync_wait <span class="version-mark">New in v5.4.0</span>
 
-> **WARNING:**
+> **Warning:**
 >
 > Currently, synchronously loading statistics is an experimental feature. It is not recommended that you use it in production environments.
 
-- Scope: GLOBAL
-- Default value: `OFF`
-- This variable controls how TiDB behaves when the waiting time of SQL optimization reaches the timeout to synchronously load complete column statistics. The default value `OFF` means that SQL execution fails after the timeout. If you set this variable to `ON`, the SQL optimization gets back to using pseudo statistics after the timeout.
+- Scope: SESSION | GLOBAL
+- Default value: `0`
+- Range: `[0, 2147483647]`
```

Comment:
Does the value range changed? from [0, 4294967295]` changed to [0, 2147483647]?

Reply:
It's not specifically changed, it was always wrong. See: https://github.com/pingcap/tidb/pull/30026 where it was introduced.

math.MaxInt32 == 32 bit signed integer (max = 2147483647).